--- php-pecl-fileinfo-1.0.1/Fileinfo-1.0.1/fileinfo.c~	2005-12-24 17:06:38.000000000 +0100
+++ php-pecl-fileinfo-1.0.1/Fileinfo-1.0.1/fileinfo.c	2005-12-24 17:06:51.828669392 +0100
@@ -103,7 +103,7 @@
 	intern = ecalloc(1, sizeof(struct finfo_object));
 	intern->zo.ce = class_type;
 	intern->zo.properties = NULL;
-	intern->std.guards = NULL;
+	intern->zo.guards = NULL;
 	intern->ptr = NULL;
 
 	retval.handle = zend_objects_store_put(intern, finfo_objects_dtor, NULL, NULL TSRMLS_CC);
